src/paulscode/android/mupen64plusae/input/map/TouchMap.java

```diff
@@ -265,6 +271,28 @@ public int getButtonPress( int xLocation, int yLocation )
     }
     
     /**
+     * Gets the frame for the N64 button with a given asset name
+     * 
+     * @param assetName The asset name for the button
+     * 
+     * @return The frame for the N64 button with the given asset name
+     * 
+     */
+    public Rect getButtonFrame( String assetName )
```

Comment:
Take a peek at the fields of the paulscode.android.mupen64plusae.util.Image class.  You might be able to just return `buttonMasks.get( i ).drawRect` or `buttonMasks.get( i ).drawable.getBounds()`.